ldms/src/ldmsd/ldmsd.h

```diff
@@ -948,54 +952,64 @@ int process_config_str(char *config_str, int *lno, int trust);
 
 char *process_yaml_config_file(const char *path, const char *dname);
 
-
+/* These flags populate the 'flags' field in the plugin structure */
+#define LDMSD_PLUGIN_SINGLE_INSTANCE 0
```

Comment:
If the there is going to be a generic "flags" field capable of having multiple bits set, it doesn't make sense to have the "LDMSD_PLUG_SINGLE_INSTANCE" define. Single instance means the absence of the LDMSD_PLUG_MULTI_INSTANCE bit, not necessarily that all bits in "flags" are zero.

Reply:
It's there for clarity (documentation), but yes it will likely never be used.

Reply:
I found that define a lot less than clear when I read it. The documentation literally says "The flags populate..." implying that these flags should be used. But 0 isn't a flag. So it shouldn't actually be used. And you made me think through that for no reason. :) So for clarity I think we should _remove_ that non-flag, define-that-should-not-be-used, and use comments for documentation instead.